tests/test_verifylib.py

```diff
@@ -144,7 +144,18 @@ def test_inspection_fail_with_non_zero_retval(self):
         }]
     })
     with self.assertRaises(BadReturnValueError):
-      run_all_inspections(layout)
+      run_all_inspections(layout, True)
+
+  def test_inspection_persistence_false(self):
+    """Test metadata link file non-persistence"""
+    os.remove("touch-bar.link")
```

Comment:
It's a good idea to make sure that there is no `touch-bar.link` before running the inspections, but blindly removing it if it indeed does not exist fails the tests. I suggest to just assert that it does not exist.